staking/effective/eligible.go

```diff
@@ -1,10 +1,19 @@
 package effective
 
-import (
-	staking "github.com/harmony-one/harmony/staking/types"
-)
+// Eligibility ..
+type Eligibility byte
 
-// IsEligibleForEPOSAuction ..
-func IsEligibleForEPOSAuction(v *staking.ValidatorWrapper) bool {
-	return v.Active && !v.Banned
-}
+const (
+	// Active means allowed in epos auction
+	Active Eligibility = iota
```

Comment:
Will 0 byte work in the case of no-op EditValidator. Basically how to differentiate no-op vs setting to Active?

Reply:
thought about that, just addressed: 6c19abe8ff689958c7ac6a697b405408b33c75a9